cli/ts/commands/deployPoll.ts

```diff
@@ -43,38 +51,56 @@ export const deployPoll = async (
     logError("Please provide a VkRegistry contract address");
   }
 
-  const vkRegistry = vkRegistryAddress ? vkRegistryAddress : vkRegistryContractAddress;
+  const vkRegistry = vkRegistryAddress || vkRegistryContractAddress;
```

Comment:
should it be `??` here instead of `||`?
Since what we wanna do is that if `vkRegistryAddress` is not undefined or null, `vkRegistry` should be assigned to `vkRegistryAddress`, else `vkRegistryContractAddress`. For using `||` the code is not intuitive, and if `vkRegistryAddress` is 0 (although it's impossible to happen), I think the `vkRegistry` would still be assigned to `vkRegistryContractAddress` --> what I mean here is that I think we only wanna include the case that `vkRegistryAddress` is `null` or `undefined`.

Reply:
I know that `vkRegistry = vkRegistryAddress ? vkRegistryAddress : vkRegistryContractAddress` is totally equal to `vkRegistry = vkRegistryAddress || vkRegistryContractAddress`, just wanna ask your opinions toward this?